src/modules/legacy/components/AccountsComparison/index.js

```diff
@@ -16,7 +16,12 @@ const AccountsComparison = ({ t }) => {
   useDeprecatedAccount();
   useSchemas();
   const wallet = useSelector(selectActiveTokenAccount);
+  const nonce = wallet.sequence?.nonce;
   const hasEnoughBalance = Number(wallet.token?.[0]?.availableBalance) >= dustThreshold;
+  const hasDepositedTokens = useMemo(
```

Comment:
Feels unjustified to use useMemo for this calculation. Using useMemo also has its performance implications, for example doing a `shouldThisUpdate` operation, which could have worse performance than checking `hasEnoughBalance && nonce >= 0`... 

I comment this because i see that we have a very generous useMemo/useCallback pattern i this codebase. useMemo and React.memo should not be thought of as guarantees of freezing a value, it is better to first not use them to make sure code works without them (very difficult bugs can appear if thinking of them as "freezing a value") and later optimize.